### ISSUE TITLE
Request: Set DNS server for AP clients

### ISSUE THREAD

**Issue by shyd** [nodemcu/nodemcu-firmware#2470]
### Missing feature
Hi, I already read #1095 and can not use the DNS proxy approach.
I also need to set the DHCP option which pushes the DNS servers to AP clients.

### Justification
In my scenario I have a ESP-12F set up as AP in a radio controlled car. My application features a web app with JQuery and Bootstrap etc. For obvious reasons those libraries will be downloaded from other CDNs rather from the ESP directly. Connecting with my Android phone I assume to have an internet connection plus being connected to the AP which does not push a default gateway so my phone uses cellular to get the libraries.
Long story short. This doesn't work, because connecting to the AP sets the DNS server of my phone to the AP's IP address.
Please give me an option to alter this DNS IP address to e.g. 8.8.8.8 or sth. else. Or just an option to disable to get a DNS address pushed at all.

### Workarounds
There is no considerable workaround.
